### ISSUE TITLE
Checkout Screen: Change Discount Verbiage to refer to Credits

### ISSUE THREAD

**Issue by ccwalburn** [Automattic/wp-calypso#68278]
### What

Change the verbiage on the Checkout to make it clear that the discount applied is the Plan credits seen on the Plans page. Right now it looks like this:

![Checkout Discount](https://user-images.githubusercontent.com/32983813/192299390-7b7dac13-042b-44a5-8499-069deea98c18.png)

### Why

The verbiage on the Checkout screen does not make it clear that the discount received is the credits that are offered on the Plans page. It appears to be some other type of promotional discount so some users are looking for ways to add their credits to the checkout screen.

### How

The Plans page says, "You have $96.00 of pro-rated credits available from your current plan.", so I would suggest using similar verbiage. Perhaps change from "Discount for first year" to "Pro-rated credits applied for first year". 

36043375-hc

**Comment by zachspears** [comment]
I strongly agree with this recommendation and have received several chats with this confusion. Marking Normal Priority due to potential high impact for resolving confusion around pro-rated discount.

**Comment by DavidRothstein** [comment]
I don't think we want to use "credits" in checkout for this, since they aren't actually credits (and it would be confusing if the user had real credits on top of this).

Probably the plans page should change here, and that's being discussed in pdgrnI-24d-p2.  Just in case it winds up using different terminology altogether, though, checkout should probably be changed at the same time.

**Comment by DavidRothstein** [comment]
>and it would be confusing if the user had real credits on top of this

Apparently it was in fact confusing when we did this in the past: See the screenshot in step 4 of p2MSmN-78a-p2#comment-35547 (note the double use of the "credit" wording) and see the discussion in that P2.

**Comment by zdenys** [comment]
>oh that's not really clear at all on the Plans page...and I wouldn't have seen that support page in the middle of purchasing a plan

6590986-hc
